bibliopixel/drivers/PiWS281X.py

```diff
@@ -0,0 +1,52 @@
+# Original code by msurguy: https://github.com/ManiacalLabs/BiblioPixel/issues/51#issuecomment-228662943
+from . channel_order import ChannelOrder
+from . driver_base import DriverBase
+from .. import log
+from .. import gamma
+try:
+    from neopixel import Adafruit_NeoPixel, Color as NeoColor
+except:
+    log.error("""PiWS281X Requires the rpi_ws281x C extension.
```

Comment:
Hard to read, consider extracting the comment as a separate `ERROR` constant ("don't mix code and data")